demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -162,6 +165,61 @@ def is_valid_fromversion_on_modified(self) -> bool:
 
         return True
 
+    @error_codes("BC107")
+    def is_valid_toversion_on_modified(self) -> bool:
+        """Check that the toversion property was not changed
+        on existing Content files.
+
+        Returns:
+            (bool): Whether the files' fromversion as been modified or not.
+        """
+        if not self.old_file:
+            return True
+
+        to_version_new = self.current_file.get(
+            "toversion"
+        ) or self.current_file.get("toVersion")
+        to_version_old = self.old_file.get("toversion") or self.old_file.get(
+            "toVersion"
+        )
+        if to_version_old != to_version_new:
```

Comment:
do we want to allow cases where the new `to_version` is higher than the previous one? 🤔 (not sure)

Reply:
ummm its make sense to allow this.

Reply:
there is another validation  'are_fromversion_toversion_synchronized' that check that toversion>fromversion